test/e2e/apps/job.go

```diff
@@ -57,6 +57,30 @@ var _ = SIGDescribe("Job", func() {
 		}
 	})
 
+	/*
+		Testcase: Ensure that the pods associated with the job are removed once the job is deleted
+		Description: Create a job and ensure the associated pod count is equal to paralellism count. Delete the job and
+						ensure if the pods associated with the job have been removed
+	*/
+	ginkgo.It("should remove pods when job is deleted", func() {
+		ginkgo.By("Creating a job")
+		job := jobutil.NewTestJob("notTerminate", "all-pods-removed", v1.RestartPolicyNever, parallelism, completions, nil, backoffLimit)
+		job, err := jobutil.CreateJob(f.ClientSet, f.Namespace.Name, job)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred(), "failed to create job in namespace: %s", f.Namespace.Name)
```

Comment:
you this instead:
https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/util.go#L2063

Reply:
Yes, right.
Now `hack/verify-code-test.sh` enforces to use the function instead of `gomega.Expect(err).NotTo(gomega.HaveOccurred())`, so we need to replace here.